Gemfile

```diff
@@ -192,14 +201,22 @@ ENV['DATABASE_ADAPTER'] ||=
     'mysql2'
   end
 
-if_true(ENV['DATABASE_ADAPTER'].strip == 'postgresql') do
-  gem 'pg', '~> 0.18.3'
+if_true(ENV['DATABASE_ADAPTER'].strip == 'postgresql' || !!ENV['INSTALL_ALL_DBS']) do
```

Comment:
I haven't checked to any great level of detail, but I think it was pretty minimal, maybe a few mb? The main difference comes from needing all of the dev libraries for each DB to compile them, but initial testing seems to imply that we can remove them again once they're compiled, so long as we have the runtime libs installed (much smaller)

Yeah, can definitely change it to `INSTALL_ALL_DATABASE_ADAPTERS`.